pkg/processor/native/processor_test.go

```diff
@@ -83,6 +83,39 @@ func (s *BildProcessorSuite) TestBildProcessor_Grayscale() {
 	assert.EqualValues(s.T(), actual, expected)
 }
 
+func (s *BildProcessorSuite) TestBildProcessor_Blur() {
+	var actual, expected []byte
+	var err error
+	cases := []struct {
+		radius float64
+		expectedFile string
+	}{
+		{
+			radius: 0.0,
+			expectedFile: "_testdata/test.jpg",
+		},
+		{
+			radius: 1.0,
+			expectedFile: "_testdata/test_blurred_1.jpg",
+		},
+		{
+			radius: 60.0,
```

Comment:
would be nice to test extreme cases, change this to the upper bound

Reply:
We can also time it here and log the value, WDYT @adalberht?

Reply:
yes that would be nice but my concern is do we have kind of file to store the value instead of checking log so that if there's new changes we can compare it to the previous result?

Reply:
Ideally, this should be handled with the help of benchmark tests.

Reply:
I think 60 is fine for now because a value of 1000 will most likely stall the CI.
We should also wrap the `manipulator.Process` call inside a goroutine and terminate it if it fails to return a result within a timeout limit.

Reply:
ok, sounds good to me. Agreed